### PR TITLE
Fix for Content field focus bug

### PR DESCRIPTION
**Pull request by zamkevich**
## Reproduce
Create two lists with a Relationship ref and Content fields with identical names. 

```
keystone.createList('foo', {
  fields: {
    ref: {
      type: Relationship,
      ref: 'bar.ref'
    },
    text: { type: Content },
  }
});
keystone.createList('bar', {
  fields: {    
    ref: {
      type: Relationship,
      ref: 'foo.ref'
    },
    text: { type: Content },
  }
});
```

Create a new doc from admin UI , then create a second doc through the Relationship field, and try to edit the Content field - you can't.

screencast 2020-06-04 17-05-40

## Problem
The min-height of the text block is specified through the wrapper. The height of the text block remains unchanged. Focus is set on the text block through a function that after by click on wrapper searches for an element by id. Focus is not set if two identical blocks are in because the id is not unique.

## Decision
Remove the wrapper and set the min-height directly in the text block.

screencast 2020-06-04 17-10-29

### CODE REVIEW

**Comment by Vultraz** [comment]
I had a similar change to this in #2510, want to check how it compares.

**Comment by zamkevich** [comment]
> I had a similar change to this in #2510, want to check how it compares.

i see, you change it all

**Comment by Vultraz** [comment]
Well that PR still has a way to go before getting merged. I could split that fix out.

**Comment by zamkevich** [comment]
> Well that PR still has a way to go before getting merged. I could split that fix out.

Actually I found one small issue: field padding is reduce click area. And now i don't know have is sense to fix it.

**Comment by Vultraz** [comment]
I'm pretty sure I ran into that in my PR and resolved it.

**Comment by zamkevich** [comment]
Anyway, i fixed it 😄

**Comment by Vultraz** [comment]
I just realized merging this would cause conflicts with 2510... oof

**Comment by zamkevich** [comment]
> I just realized merging this would cause conflicts with 2510... oof

I see that you completely rewrite the block of code that I corrected in the /src/views/Field.js
and deleted second file /src/views/editor/index.js

If you merge your PR in the near future, then mine does not make sense.

But if you fix it a couple more months, then it makes sense to fix the bug right now.

**Comment by zamkevich** [comment]
I really can't understand why prietter fails simple tests.

image

File: packages/field-content/src/views/editor/index.js [hunk 1]
```diff
@@ -76,6 +76,10 @@ function Stories({ value: editorState, onChange, blocks, className, id, isDisabl
           onChange(value);
         }}
         readOnly={isDisabled}
+        css={{
+          minHeight: 200,
+          padding: '16px 32px',
+        }}
```
Comment (MadeByMike) [inline packages/field-content/src/views/editor/index.js:82]: Hmmm, why was this added?

Comment (MadeByMike) [reply packages/field-content/src/views/editor/index.js:82]: Oh ok matches editor CSS - no worries :)

**Comment by MadeByMike** [comment]
Ok, can we please check that focusing the editor with both the Keyboard and Pointer works?

Also @zamkevich can you please run `yarn changeset` in the root directory. Follow the prompts and skip both major and minor to add a `patch` changeset in this case. Description of changes should be full sentence with past tense. Ed. "Fixed a bug with... ."

**Comment by zamkevich** [comment]
@MadeByMike Keyboard focus also works fine. 

screencast 2020-06-05 14-23-12

You can use my repository for tests. That already test configuration, and simple readme.
https://github.com/zamkevich/keystone-tests/

File: .changeset/odd-mirrors-grin.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@keystonejs/field-content': patch
+---
+
+Fixed a bug with pointer focus on fields that have identical names
```
Comment (MadeByMike) [inline .changeset/odd-mirrors-grin.md:5]:
```suggestion
Fixed a bug with pointer focus on fields that have identical names.
```

**Comment by MadeByMike** [comment]
@all-contributors please add @zamkevich for code
